pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md

```diff
@@ -0,0 +1,444 @@
+
+> 💡 Tip : If you are familiar with [Docker](https://www.docker.com/){.external} or [Kubernetes](https://kubernetes.io/){.external}, you can use the psql tool inside a postgres Docker [image](https://hub.docker.com/_/postgres){.external} instead of installing the tool locally.
+>
+> Examples :
+> ```sh
+> $ kubectl run postgres-client --image=postgres:latest -it --rm --restart=Never -- /bin/bash
+> ```
+> or
+> ```
+> $ docker run -it --rm postgres /bin/bash
+> ```
+>
+
+#### Connect to the database
+
+The useful parameters are:
+
+- The db Host, the db Port
+    - Get them from the "General Information tab"
+
+- The db Name
+    - Get it from the "Databases" tab, usually "defaultdb"
+
+- The db User
+    - Get it from the "Users" tab, usually "avnadmin"
+
+- The db Password
+    - Get it after you reset it.
+
+![Password reseted](images/postgresql-tuto-01-connect-strapi-to-managed-postgresql11.png){.thumbnail}
+
+Now connect to the database with the following command
+
+```sh
+psql --dbname=defaultdb  --host=postgresql-xxxxxxxx-xxxxxxxxx.database.cloud.ovh.net --port=20184 --username=avnadmin  --password 
+```
+Enter the password and press Enter
+
+<pre class="console"><code>$ psql --dbname=defaultdb  --host=postgresql-xxxxxxxx-xxxxxxxxx.database.cloud.ovh.net --port=20184 --username=avnadmin  --password
+Password: 
+psql (14.1 (Debian 14.1-1.pgdg110+1))
+SSL connection (protocol: TLSv1.3, cipher: TLS_AES_256_GCM_SHA384, bits: 256, compression: off)
+Type "help" for help.
+
+defaultdb=> 
+</code></pre>
+
+Setup is done, your Managed PostgreSQL database is fully operational, let's go further and use it with <img src="images/strapi-logo-monogram-purple.png" alt="strapi logo" width="12px"> [Strapi](https://strapi.io/){.external}.
+
+### Before beginning, repare your environment
+
+What you need here is to check your nodejs and yarn versions.
+According to the official documention on [github](https://github.com/strapi/strapi){.external}, check that your existent nodejs installation matches with "NodeJS >= 12 <= 16".
+
+❗ No specific version of yarn is specified, this tutorial was made with the 1.22.17 one.
+You can follow this different tuorials if you need to install some of the tools:
+- [nodejs](https://nodejs.org/en/download/package-manager/){.external}
+- [nvm](https://github.com/nvm-sh/nvm){.external}
+- [yarn](https://classic.yarnpkg.com/lang/en/docs/install/){.external}
+
+### Declare the nodejs version
+
+We want to use the latest LTS version of nodejs, so inside your development environment, use this command :
+```
+$ nvm use 16.13.1
+```
+<pre class="console"><code>$ nvm use 16.13.1
+Now using node v16.13.1 (npm v8.1.2)
+</code></pre>
+
+### Setup a new instance of Strapi
+
+Strapi offers a simple way to install his product, with yarn.
+Run the following command in a terminal:
+```
+yarn create strapi-app my-strapi
+```
+
+You will be prompted to choose between the Quickstart or Custom method, select Custom and press Enter.
```

Comment:
select `Custom`{.action} and press `Enter`{.action}.

Reply:
Done